Eplant/views/PublicationViewer/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { styled, Tab, Tabs, Typography } from '@mui/material'
+import * as React from 'react'
+import { ViewProps } from '../View'
+import { GeneRIFs } from './GeneRIFs'
+import { Publications } from './Publications'
+import { PublicationViewerData, TabValues } from './types'
+
+const SecondaryColour = styled('span')(({ theme }) => ({
+  color: theme.palette.text.secondary,
+}))
+
+export const PublicationViewer = {
+  name: 'Publication Viewer',
+  dataType: PublicationViewerData,
+  component({ geneticElement, activeData }: ViewProps<PublicationViewerData>) {
+    const [tab, setTab] = React.useState<TabValues>('publications')
+    return (
+      <div>
+        <Typography variant='h4'>
+          {geneticElement.id} <SecondaryColour>I</SecondaryColour> {geneticElement.aliases.join(', ')}
```

Comment:
It might look better to use something like `<Divider orientation="vertical" variant="middle" />` here instead of the `I`.